lib/rust/parser/tests/parse.rs

```diff
@@ -471,6 +471,91 @@ fn import() {
     cases.into_iter().for_each(|(code, expected)| test(code, expected));
 }
 
+#[test]
+fn export() {
+    #[rustfmt::skip]
+    let cases = [
+        ("export Foo", block![(Export () () ((Ident export) (Ident Foo)) () ())]),
```

Comment:
`export Foo` isn't valid in current parser.

Reply:
I see. In the next PR I'll change it to `prj.Data.Foo`.